libraries/AP_HAL_PX4/Util.cpp

```diff
@@ -183,6 +183,9 @@ void PX4Util::perf_count(perf_counter_t h)
 void PX4Util::set_imu_temp(float current)
 {
     if (!_heater.target || *_heater.target == -1) {
+#if defined(CONFIG_ARCH_BOARD_PX4FMU_V3)
```

Comment:
@proficnc and I went around a bit on that.

People are copying parameters from PixHawk1s to PixHawk2s and frying their boards.  Since they're likely to thus copy their board type parameter....